compiler/rustc_codegen_llvm/src/builder.rs

```diff
@@ -31,12 +31,135 @@ use tracing::{debug, instrument};
 use crate::abi::FnAbiLlvmExt;
 use crate::attributes;
 use crate::common::Funclet;
-use crate::context::CodegenCx;
+use crate::context::{CodegenCx, SimpleCx};
 use crate::llvm::{self, AtomicOrdering, AtomicRmwBinOp, BasicBlock, False, True};
 use crate::type_::Type;
 use crate::type_of::LayoutLlvmExt;
 use crate::value::Value;
 
+// All Builders must have an llfn associated with them
```

Comment:
That's not quite right, is it? Probably not even right for the normal builder

Reply:
yep, I'll adjust it.